projects/packages/waf/actions.php

```diff
@@ -20,6 +28,7 @@
 add_action(
 	'plugin_loaded',
 	function () {
+		Waf_Runner::update();
```

Comment:
No, what happens is it checks the hardcoded WAF version and compares it against the WAF version stored in the site db. If they don't match, then it updates the rules.php file again. 

The naming here is a relic from when I believed we could just run it on an update hook (since the WAF is a package, we can't - we're tied to Jetpack updates). So I'll ditch the update() method, and replace it with a has_version_changed() method, then call generate_rules() if it has. That would be more explicit and accurate naming-wise.